axelor-account/src/main/java/com/axelor/apps/account/web/YearController.java

```diff
@@ -40,7 +41,7 @@ public void close(ActionRequest request, ActionResponse response) {
 		catch(Exception e)  { TraceBackService.trace(response, e); }	
 	}
 	
-	public void generatePeriods(ActionRequest request, ActionResponse response) {
+	public void generatePeriods(ActionRequest request, ActionResponse response) throws AxelorException {
```

Comment:
Should have a Try catch statement here and store the exception using TraceBackService.